server/feeders/restaurantfeeder.js

```diff
@@ -85,15 +85,15 @@ var feedOrionRestaurants = function() {
           utils.sendRequest('POST', attributes, null, fwHeaders)
```

Comment:
There is an upcoming PR for the feeders to adapt it to the CLI, so I'll address all the comments for the feeders in another PR